tests/acceptance/cleanup-test.js

```diff
@@ -0,0 +1,19 @@
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { visit, currentURL } from '@ember/test-helpers';
+import setupLifelineValidation from 'ember-lifeline/test-support';
+
+module('Acceptance | clean up', hooks => {
```

Comment:
Can we change this from arrow to function? I know it's a bit nit picky, but it's inline with the rest of our tests. Also, it adheres to this rule -> https://github.com/platinumazure/eslint-plugin-qunit/blob/master/lib/rules/no-arrow-tests.js, which I happen to agree with.

Reply:
i'm all about the linting 👍. do you want me to add eslint to the project with that rule too? Not sure if addons are supposed to have their own rules.